src/helpers/api.js

```diff
@@ -0,0 +1,10 @@
+export function checkStatus(response) {
+    const { status, data } = response
+  
+    if (status != 200) {
+        return Promise.reject(data)
+    }
+  
+    return Promise.resolve(data)
+  }
```

Comment:
I'm intrigued. Curious to know what this does for you that axios doesn't. Is it just to unwrap the nested `data` key?

Reply:
I usually use that handler for native fetch, I forget axios handles this for you out of the box. I've removed it in my refactor. Thanks for catching that.